### PR TITLE
added fun fact

### PR DESCRIPTION
**Pull request by reherr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,3 +102,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 | Name                 | Fun Fact |
 | -------------------- | --- |
+| Rebecca Herrera | If the Earth doubled in size, tress would immediately fall down. |
```
Comment (SaraDawner2000) [inline README.md:105]: cool!

**Review by SaraDawner2000 — APPROVED** [review]
Looks great! 😊

**Review by rezahusain — APPROVED** [review]
Its alright

**Review by MayaS1111 — APPROVED** [review]
Looks great, learn something new every day.

**Review by FadiBaker92 — COMMENTED** [review]
looks good

**Review by rezahusain — APPROVED** [review]
Looks good! 😌🌲

**Review by borvux — APPROVED** [review]
Looks good!

**Review by FadiBaker92 — APPROVED** [review]
looks good

**Review by MayaS1111 — APPROVED** [review]
Great

**Pull request merged**